src/db/index.ts

```diff
@@ -1763,12 +1768,17 @@ export const getTxProposal = async (
  */
 export const releaseTxProposalUtxos = async (
   mysql: ServerlessMysql,
-  txProposalId: string,
+  txProposalIds: string[],
 ): Promise<void> => {
-  await mysql.query(
-    'UPDATE `tx_output` SET `tx_proposal` = NULL, `tx_proposal_index` = NULL WHERE `tx_proposal` = ?',
-    [txProposalId],
+  const result: OkPacket = await mysql.query(
+    `UPDATE \`tx_output\`
+        SET \`tx_proposal\` = NULL,
+            \`tx_proposal_index\` = NULL
+      WHERE \`tx_proposal\` IN (?)`,
+    [txProposalIds],
   );
+
+  assert.strictEqual(result.affectedRows, txProposalIds.length);
```

Comment:
Since the idea is to throw an error if the amount of updated rows is not the expected, an error message would be helpful.
```suggestion
  assert.strictEqual(result.affectedRows, txProposalIds.length, 'Not all utxos were correctly updated');
```

Reply:
Much better, thanks! Done in 2a23e8a3648c763b6646992760fda9d72da221e2